csharp/autobuilder/Semmle.Autobuild/BuildScript.cs

```diff
@@ -131,8 +132,8 @@ public override int Run(IBuildActions actions, Action<string, bool> startCallbac
         class BindBuildScript : BuildScript
         {
             readonly BuildScript s1;
-            readonly Func<IList<string>, int, BuildScript> s2a;
-            readonly Func<int, BuildScript> s2b;
+            readonly Func<IList<string>, int, BuildScript>? s2a;
+            readonly Func<int, BuildScript>? s2b;
```

Comment:
Why are these nullable? The corresponding parameters that are assigned to these fields are not.

Reply:
There are two constructors to `BindBuildScript` and they only set one of these fields. The other one is always `null`.

Reply:
Ah, right. Sorry.